db/seeds/questionnaires/mentor_post_assessment.rb

```diff
@@ -3,185 +3,42 @@
-                       title: 'Zou je nog een extra herinnering willen ontvangen op een bepaalde tijd?',
-                       options: ['Nee'],
-                       otherwise_label: 'Ja, namelijk om:'
-                     }, {
-                       id: :v25,
-                       type: :radio,
-                       title: 'Je kreeg nu elke donderdag een vragenlijst. Zou je deze liever op een andere dag krijgen?',
-                       options: ['Nee', 'Ja, op maandag', 'Ja, op dinsdag', 'Ja, op woensdag', 'Ja, op vrijdag'],
-                       section_end: true
-                     }, {
-                       section_start: 'Missen van vragenlijsten',
-                       id: :v26,
-                       type: :checkbox,
-                       title: 'Wat waren de redenen dat je wel eens een vragenlijst hebt gemist? (meerdere antwoorden mogelijk)',
-                       options: ['Ik heb nooit een vragenlijst gemist',
-                                 'Ik kreeg geen sms',
-                                 'De link naar de vragenlijst werkte niet',
-                                 'Ik had geen tijd',
-                                 'Ik had geen zin',
-                                 'Ik was het vergeten',
-                                 'Mijn batterij was leeg',
-                                 'Ik zat op dat moment niet met mijn telefoon op wifi',
-                                 'De databundel van mijn telefoon was op',
-                                 'De vragenlijst was al verlopen'
-                       ],
-                       section_end: true
-                     }, {
-                       section_start: 'Volhouden',
-                       id: :v27,
-                       type: :radio,
-                       title: 'Je hebt nu drie weken meegedaan aan dit onderzoek. Denk je dat je ditzelfde onderzoek ook voor zeven maanden zou volhouden?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, omdat:',
-                       section_end: true
-                     }, {
-                       section_start: 'Tot slot',
-                       id: :v28,
-                       type: :textarea,
-                       title: 'Wat zou jij willen verbeteren aan de webapp die je de afgelopen drie weken hebt gebruikt?',
-                       section_end: true
-                     }]
+  id: :v1,
+  type: :textarea,
+  title: 'Kan je in één woord beschrijven hoe jij het ervaart jij om jongeren te begeleiden binnen {{je_begeleidingsinitiatief}}?'
+}, {
+  id: :v2,
+  type: :range,
+  title: 'In hoeverre vind jij dat {{deze_student}} vooruitgang heeft geboekt in {{zijn_haar_student}} begeleidingstraject?',
+  foreach: :student,
+  labels: ['Helemaal niet ', 'heel veel']
+}, {
+  id: :v3,
+  type: :range,
+  title: 'Hielp het invullen van de app je om jouw jongeren beter te begeleiden? ',
+  labels: ['Helemaal niet ', 'helemaal wel']
```

Comment:
spatie aan eind van label

Reply:
En ook hoofdlettergebruik weer

Reply:
hoofdletter gebruik is wel consistent volgens mij, eerste label altijd wel, en tweede niet (ik dacht dat we dat overal zo deden), maar ik zal wel even kijken naar de andere vragenlijsten

Reply:
Nee dat doen ze niet overal zo. soms hebben ook geen van de labels hoofdletters. Plus het staat superlelijk dus ik zou het niet zo doen.

Ik zou het zo aanpassen dat geen van de labels hoofdletters heeft. Dan is het namelijk consistent met onze andere vragenlijsten. En ze krijgen sowieso de preview van deze vragenlijsten nog te zien dus dan kunnen ze er dan nog feedback opgeven, maar ik denk eigenlijk dat die hoofdlettering per ongeluk is gedaan omdat Microsoft Word automatisch een zin met een hoofdletter wil laten beginnen en dat ze er verder niet bewust een beslissing over hebben gemaakt. Dat is meer layout dan content lijkt me, en dat is aan ons.